packages/test/package.json

```diff
@@ -8,7 +8,11 @@
     "build:ts": "tsc --build",
     "build:protos": "node ./scripts/compile-proto.js",
     "test": "ava ./lib/test-*.js",
-    "test.watch": "ava --watch ./lib/test-*.js"
+    "test.watch": "ava --watch ./lib/test-*.js",
+    "test-split-one": "ava ./lib/test-integration-split-one.js",
```

Comment:
Certainly useful while developing locally, but I don't think we should commit such aliases. Too specic/volatile.